backend/erc20.go

```diff
@@ -80,6 +80,18 @@ var erc20Tokens = []erc20Token{
 		unit:  "ZRX",
 		token: erc20.NewToken("0xe41d2489571d322189246dafa5ebde1f4699f498", 18),
 	},
+	{
+		code:  "eth-erc20-wbtc",
+		name:  "Wrapped Bitcoin",
+		unit:  "WBTC",
+		token: erc20.NewToken("0x2260fac5e5542a773aa44fbcfedf7c193bc2c599", 8),
```

Comment:
please use correct capitalization here (checksum encoding):

`0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599`

Reply:
wrong again since squash? You should probably look for the previous commit before squashing in your reflog and do it again :sweat_smile: